kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/helpers/ApiHelper.java

```diff
@@ -79,40 +80,38 @@ public void createSchema(String clusterName, String schemaName, SchemaType type,
         }
     }
 
-    @SneakyThrows
     public void deleteSchema(String clusterName, String schemaName) {
         try {
             schemaApi().deleteSchema(clusterName, schemaName).block();
         } catch (WebClientResponseException ignore) {
         }
     }
 
-    @SneakyThrows
     public void deleteConnector(String clusterName, String connectName, String connectorName) {
         try {
             connectorApi().deleteConnector(clusterName, connectName, connectorName).block();
         } catch (WebClientResponseException ignore) {
         }
     }
 
```

Comment:
looks like connectorApi().deleteConnector() and connectorApi().createConnector() methods are implemented with throwing RestClientException. in this case @SneakyThrow is needed

Reply:
Agree , changed it